### PR TITLE
Addition

### PR DESCRIPTION
**Pull request by abhijit-kakade-dev**
+ Addition of two numbers.

### CODE REVIEW

File: addition.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+int main()
+{
+    int a, b, sum;
```
Comment (akshay-kakade-dev) [inline addition.c]: Never declare variables like a & b.  The variable should be declared with some meaningful names.

**Review by akshay-kakade-dev — CHANGES_REQUESTED** [review]
Please change the code according to the comments.

File: addition.c [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    printf("Enter two numbers to add");
+    scanf("%d%d", &a, &b);
+
+    sum = a + b;
+
```
Comment (akshay-kakade-dev) [inline addition.c]: Do we really need the sum here? Can we optimize it? And when should we declare a variable?

File: addition.c (hunk 1 above)
Comment (abhijit-kakade-dev) [reply addition.c]: Variable name has been changed to firstnum & secondnum

File: addition.c (hunk 2 above)
Comment (abhijit-kakade-dev) [reply addition.c]: No, it is not necessary. This variable optimized as firstnum + secondnum

**Pull request merged**